surf-for-type-safety/move/sources/message_board.move

```diff
@@ -0,0 +1,230 @@
+module message_board_addr::message_board {
+    use std::option::Option;
+    use std::string::String;
+
+    use aptos_framework::object::{Self, Object, ObjectCore, ExtendRef};
+
+    const BOARD_OBJECT_SEED: vector<u8> = b"message_board";
+
+    struct BoardObjectController has key {
+        extend_ref: ExtendRef,
+    }
+
+    struct Message has key, drop {
```

Comment:
if this is just a simple Surf example - can we simplify the contract even more? naybe a simple entry fun with a string_content and a view function to view the content. That way this example is only and mostly focuses on Surf usage

Reply:
i want to demonstrate all common types